.github/workflows/automerge.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v3
       - name: automerge-action
         id: automerge-action
-        uses: conda-forge/automerge-action@main
+        uses: conda-forge/automerge-action@master
```

Comment:
It doesn't really matter, but interestingly enough, rendering yields  `@master` for the `3.8` and `3.9` branches and `@main` for the `3.10` branch.
(I was a bit confused when I noticed that with my first local rerendering, hence querying the bot multiple times to get confirmation it really wants that change.)

Reply:
Looks like a `conda-forge.yml` change would do it:

https://github.com/conda-forge/python-feedstock/blob/main/conda-forge.yml#L12-L14

vs

https://github.com/conda-forge/python-feedstock/blob/3.8/conda-forge.yml

Might be worth syncing up some of the other stuff, as well...

Reply:
Thanks for digging, @bollwyvl!